src/core/qgshttptransaction.cpp

```diff
@@ -486,6 +497,16 @@ void QgsHttpTransaction::dataStateChanged( int state )
 }
 
 
+void QgsHttpTransaction::ignoreSslErrors( const QList<QSslError> &errors )
+{
+  Q_UNUSED( errors );
```

Comment:
this looks a bit dangerous.
Wouldn't it be better to not ignore and log them instead?

Reply:
Yes, it will be better to log the errors, but it's just a quick and dirty fix in a class removed in QGIS 3.

Reply:
But it would also be trivial to switch to logging, quick and dirty or not?

Reply:
Done